backend/python/app/services/implementations/comment_service.py

```diff
@@ -15,37 +18,81 @@ def __init__(self, logger=current_app.logger):
 
     def create_comment(self, comment):
         try:
+            # TODO: remove cast to int once get_user_id_from_request is updated
+            user_id = int(get_user_id_from_request())
             new_comment = Comment(**comment)
-            new_comment.time = datetime.utcnow()
-            new_comment.resolved = False
-            comment_index = 0
+            new_comment.user_id = user_id
 
-            max = (
-                db.session.query(func.max(Comment.comment_index))
+            story_translation = (
```

Comment:
nice query! nit: I believe ` db.session.query(StoryTranslation).join(....` could be rewritten as `StoryTranslation.query.join(...`. I think the latter option is more clear